pages/1.7/release-notes/1.7.3/index.md

```diff
@@ -0,0 +1,74 @@
+*   After providing the **Agent Private IP List**, the automated GUI installer continues to show a warning for **Master Private IP List** that you can ignore: `agent_list must be provided along with master_list`.
+*   The automated GUI installer does not validate whether there are duplicate IPs in the **Master Private IP List** and **Agent Private IP List** until you go back and re-click in the **Master Private IP List** window.
+*   If you stop and restart the automated GUI installer after running pre-flight, the setup page will not show you which IP detect script was selected.
+*   The automated installer only provisions private agents. To install public agents please see the [documentation][10].
+*   Occasionally the system health backend might panic and exit because of [this bug][11] in godbus library.
+*   You can sort system health by systemd unit. However, this search can bring up misleading information as the service itself can be healthy but the node on which it runs is not. This manifests itself as a service showing "healthy" but nodes associated with that service as "unhealthy". Some people find this behavior confusing.
+*   The system health API relies on Mesos DNS to know about all the cluster hosts. It finds these hosts by combining a query from `mesos.master` A records as well as `leader.mesos:5050/slaves` to get the complete list of hosts in the cluster. This system has a known bug where an agent will not show up in the list returned from `leader.mesos:5050/slaves` if the Mesos slave service is not healthy. This means the system health API will not show this host. If you experience this behavior it's most likely your Mesos slave service on the missing host is unhealthy.
+*   Beginning with the next release of DC/OS the `slave_public` role for public agents is changing to `agent_public`.
+
+**DC/OS Marathon**
+
+*   **Persistent local volumes** With Docker, the containerPath must be relative and will always appear in `/mnt/mesos/sandbox/`. If your application (e.g. a DB) needs an absolute directory this won’t work.
+
+    *   Volume cleanup [MESOS-2408][12]
+    *   If you go above your quota, your task will be killed and that task can never recover.
+
+*   **External/network volumes** No RO access from multiple tasks [emccode/dvdcli/issues/15][13]
+
+*   **Authorization** - In this release we have perimeter security & auth, but not internal auth. Requests originating in the cluster - i.e. that don’t have an auth token issued by AdminRouter - are not subject to authorization. Example: Marathon-LB running on DC/OS will work as expected against a Marathon with Security Plugin enabled: It will see all apps despite not having authentication credentials. [enterprise type="inline" size="small"/]
+
+See additional known issues at <a href="https://support.mesosphere.com" target="_blank">support.mesosphere.com</a>.
+
+## <a name="1-7-3"></a>1.7.3 - July 27, 2016
```

Comment:
List this section before the general info about 1.7 (before the Known Issues section).